src/components/header/HomeHeader.vue

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+import { ref, watch } from 'vue'
+import { useDebounce } from '../../custom-hooks/use-debounce'
+import { useParamsStore } from '../../stores/params-store'
+import { defineComponent } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+export default defineComponent({
+  setup() {
+    const paramsStore = useParamsStore()
+    const searchQuery = ref(paramsStore.searchQuery)
+    const debouncedSearchQuery = useDebounce(searchQuery, 1000)
+    const { t } = useI18n()
+    watch(debouncedSearchQuery, (newValue) => {            
+      paramsStore.setSearchQuery(newValue)
+    })
```

Comment:
```suggestion
    const { t } = useI18n()
    const paramsStore = useParamsStore()
    const searchQuery = ref(paramsStore.searchQuery)
    const debouncedSearchQuery = useDebounce(searchQuery, 1000)
    
    watch(debouncedSearchQuery, (newValue) => {            
      paramsStore.setSearchQuery(newValue)
    })
```
Đưa t lên đầu tiên, thêm line trống giữa các biến và watch

Reply:
done